src/platform/endpoint/common/endpointProvider.ts

```diff
@@ -51,6 +52,7 @@ type ICompletionsModelCapabilities = {
 export interface IModelAPIResponse {
 	id: string;
 	name: string;
+	urlOrRequestMetadata?: string | RequestMetadata;
```

Comment:
Should we extend this interface so it's not appearing as if this is something that is returned by CAPI?